test/ray_julia_jll/gcs_client.jl

```diff
@@ -1,44 +1,35 @@
 @testset "GCS client" begin
     using UUIDs
     using .ray_julia_jll: JuliaGcsClient, Connect, Put, Get, Keys, Exists, Status, ok,
-                          ToString
+                          ToString, Disconnect
 
     client = JuliaGcsClient("127.0.0.1:6379")
```

Comment:
Add a test for calling `Disconnect` on non-connected clients

Reply:
huh, segfault.. I'll look into this

Reply:
needed to protect Disconnecting in the wrapper code